### PR TITLE
Change the huggingface space homepage.

### PR DESCRIPTION
**Pull request by nsthorat**
- Also disable add concept when auth is enabled.

image

https://huggingface.co/spaces/lilacai/nikhil_staging

### CODE REVIEW

File: web/blueprint/src/lib/components/HuggingFaceSpaceWelcome.svelte [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    >
+      <p class="text-sm">
+        Try the Lilac dataset viewer on the the pre-loaded <a href={tryDataset.originalLink}
+          >{tryDataset.displayName}</a
+        > dataset.
+      </p>
+    </WelcomeBanner>
+  {/if}
+
+  <WelcomeBanner
```
Comment (dsmilkov) [inline web/blueprint/src/lib/components/HuggingFaceSpaceWelcome.svelte:68]: would it make sense to change the background color slightly on hover (to show it's like a button?)

File: web/blueprint/src/lib/components/Navigation.svelte [hunk 2]
```diff
@@ -108,8 +110,14 @@
     </div>
   </NavigationGroup>
   <NavigationGroup title="Concepts" tagGroups={taggedConcepts} isFetching={$concepts.isFetching}>
-    <div slot="add" class="w-full">
+    <div
+      slot="add"
+      class="w-full"
+      use:hoverTooltip={{text: !canCreateConcepts ? 'Login to create a concept.' : undefined}}
```
Comment (dsmilkov) [inline web/blueprint/src/lib/components/Navigation.svelte:116]: for whatever reason, I don't see the login button in the top right corner anymore

**Review by dsmilkov — APPROVED** [review]

**Comment by dsmilkov** [comment]
awesome work!

File: web/blueprint/src/lib/components/Navigation.svelte (hunk 2 above)
Comment (nsthorat) [reply web/blueprint/src/lib/components/Navigation.svelte:116]: this was orthogonal to the space existing, but I fixed it so if you can create a dataset then GettingStarted shows up.

File: web/blueprint/src/lib/components/HuggingFaceSpaceWelcome.svelte (hunk 1 above)
Comment (nsthorat) [reply web/blueprint/src/lib/components/HuggingFaceSpaceWelcome.svelte:68]: done

**Pull request merged**